### DISCUSSION TITLE
Deploy to subfolder

### DISCUSSION THREAD

**Discussion by TheOneWithTheBraid** [JamesIves/github-pages-deploy-action#636] [Q&A]
I would love the idea to deploy to subfolders in the given branch.

In my case e.g. I deploy automated builds for simple testing. I would like to have the option to deploy to a subfolder based on my branch name. This way I could access up-to-date build of my applications for every branch / MR.

Imaging: I push a branch `new-feature` to my project `username/unicorn-project`. This way, I could simply access my built code by https://username.github.io/unicorn-project/new-feature/

What do you think of this?

**Comment by JamesIves** [comment]
Take a look at the target-folder input, I believe that's what you're looking for. You can find an example in the project README.
